webui/react/src/routes/index.ts

```diff
@@ -125,6 +127,6 @@ export const routeAll = (path: string): void => {
   if (!isReactRoute(path)) {
     routeToExternalUrl(path);
   } else {
-    history.push(path);
+    history.push(path, { loginRedirect: clone(window.location) });
```

Comment:
hmm is this needed? routeAll isn't dedicated for sign in use. thinking how I could test if this is needed.

from the signIn routeAll is only triggered after everything is done :thinking:  

the other use case seems to be from the logout button (Link) where there we don't care about the redirect.